include/soci/column-info.h

```diff
@@ -89,11 +88,19 @@ struct type_conversion<column_info>
             ci.type = dt_string;
             ci.dataType = db_string;
         }
-        else if (type_name == "integer" || type_name == "INTEGER")
+        else if (type_name == "integer" || type_name == "INTEGER" ||
+		 type_name == "int" || type_name == "INT" ||
+		 type_name == "bigint" || type_name == "BIGINT")
```

Comment:
Shouldn't this map to `db_int64`?

Reply:
I've mapped this correctly on my side - I'll commit it in the next few days.